src/components/posts/PostDetail.js

```diff
@@ -26,7 +27,11 @@ export const PostDetail = ({token}) => {
         <div className="buttons">
             <button
             onClick={
-                ()=>{}
+                (e)=>{
+                    e.preventDefault()
+                    deletePosts(postId)
```

Comment:
I think this feature should be in the myposts module. Not sure it belongs here in the post details.